### ISSUE TITLE
[Editorial] References to numeric codepoints in prose: consistency

### ISSUE THREAD

**Issue by michaelhkay** [qt4cg/qtspecs#1165]
A quick glance in F+O finds:

```
 \n (newline, x0A)
π (x3C0)
the glyph ≂̸ which is expressed using the two codepoints #x2242 #x0338
A format token of &#x661; (Arabic-Indic digit one)
the format token &#x2460; (circled digit one, ①)
the actual Unicode character COMBINING DIARESIS (Unicode codepoint U+0308) or &#x0308;
The Latin small letter dotless i (ı, U+0131, used in Turkish)
the Unicode replacement character (U+FFFD)
CRLF (U+000D, U+000A), LF (U+000A), or CR (U+000D)
comma "," (U+002C)
the Unicode quotation mark " (U+0022)
a single newline (U+000A) character
```

I feel we could do better...

**Comment by michaelhkay** [comment]
The conventions used by Unicode are defined at https://www.unicode.org/versions/Unicode15.0.0/appA.pdf

This would suggest using U+xxxxxx where xxxxxx is between 4 and 6 hex digits, followed if necessary by the Unicode character name in small caps.

Perhaps we should use `<char>002C</char>` in the XML, and expand it to `U+0026 (COMMA, ",")` in the stylesheets? A good use case for the new `fn:ucd` function!

**Comment by ndw** [comment]
That seems like a good idea. I've always been torn in XML specifications about whether it's better to use the Unicode U+xxxx standard or the numeric character references. Your survey suggests we've already got a bias towards the Unicode standard, so being consistent seems reasonable.

**Comment by michaelhkay** [comment]
Fixed by #1184
